tools/tpm2_getmanufec.c

```diff
@@ -447,7 +447,7 @@ int RetrieveEndorsementCredentials(char *b64h)
         goto out_easy_cleanup;
     }
 
-    rc = curl_easy_setopt(curl, CURLOPT_VERBOSE, respfile);
+    rc = curl_easy_setopt(curl, CURLOPT_VERBOSE, 0L);
```

Comment:
Yeah looking at this again, we want a verbose log file if respfile is set. So either !!respFile to change it to an int type, or respfile != NULL, which will do the same thing.